### PR TITLE
Add docs.

### PR DESCRIPTION
**Pull request by ry**
Ref https://github.com/denoland/deno/pull/1012

### CODE REVIEW

File: Docs.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+### Example: An implementation of the unix "cat" program
+
+The copy here is actualy zero-copy. That is, it reads data from the socket and
+writes it back to it without ever calling a memcpy() or similiar.
+
+```ts
+import * as deno from "deno";
+
+for (let i = 1; i < deno.argv.length; i++) {
```
Comment (J2P) [inline Docs.md]: Do not you need to replace `argv` with `args`?

Comment (J2P) [reply Docs.md]: Thank you.

**Comment by GrosSacASac** [comment]
I am interested what kind of API is supported.

TCP yes
HTTP ?
WebSocket ?

Could be in the readme

**Comment by kitsonk** [comment]
@GrosSacASac for now, it is `deno --types`.  We will generate more static documents based on that information in the future (hopefully near future).  See #573.

**Pull request merged**